### ISSUE TITLE
go.obo uploaded on 01-Dec-2019

### ISSUE THREAD

**Issue by nicolasfredesfranco** [bio-ontology-research-group/deepgoplus#38]
Is the go.obo file uploaded on the data web page on 01-Dec-2019 the GO version of 2019? Are the train_data.pkl and test_data.pkl files, uploaded on 02-Dec-2019, generated from this go.obo (2019) and the UniProt version of 2019?

**Comment by coolmaksat** [comment]
Yes, if you look at the first lines in go.obo file you can find the release data which says "releases/2019-01-26".
The train and test data should use this version, but I'm not sure. Better to use the archived files. For the data generated
in 2019, there is data-2019.tar.gz file.
